### ISSUE TITLE
Constraints on Data Download Capacity?

### ISSUE THREAD

**Issue by jack-lundquist** [chriswhong/plutoplus#25]
Using both the square-drawing and map-view selection methods, I tried to download all pluto data for the 5 boroughs and ended up with an incomplete dataset of lots.

**Comment by chriswhong** [comment]
Obviously, the preferred method of getting "everything" is to get the raw PLUTO dataset from the Department of City Planning.  However, we should determine what a reasonable limit is to the selected geographic area, and provide some kind of warning that the dataset might be incomplete.
